Formula/mysql-connector-c++.rb

```diff
@@ -1,9 +1,3 @@
-class MysqlClientRequirement < Requirement
-  fatal true
-  default_formula "mysql"
-  satisfy { which "mysql" }
```

Comment:
True, but this formula really only needs `mysql_config`. It does not need the full-blown mysql server, just a way to connect to one. I used the example [here](https://dev.mysql.com/doc/connector-cpp/en/connector-cpp-examples-complete-example-1.html) (with some minor modifications) to test this.